libs/barista-components/filter-field/src/filter-field.spec.ts

```diff
@@ -43,25 +43,100 @@ import {
   FILTER_FIELD_TEST_DATA_SINGLE_DISTINCT,
   FILTER_FIELD_TEST_DATA_SINGLE_OPTION,
 } from '@dynatrace/testing/fixtures';
-import { TEST_DATA_EDITMODE_ASYNC } from './filter-field-edit-mode.spec';
 import {
-  TEST_DATA_SUGGESTIONS,
-  TEST_DATA_EDITMODE,
```

Comment:
Why did you move those fixtures into this file?

Reply:
My fault. It was done some time ago and then I found that you separated the files and tried to keep up to date. I'll put it back